docs/reference/commandline/create.md

```diff
@@ -98,6 +98,8 @@ Options:
       --privileged                    Give extended privileges to this container
   -p, --publish value                 Publish a container's port(s) to the host (default [])
   -P, --publish-all                   Publish all exposed ports to random ports
+      --pull string                   Pull image before creating ("always"|"missing"|"never") (default "missing")
```

Comment:
Thanks for this one; while reviewing your PR, I noticed that we missed adding this to the docs. Given that the `--pull` option is already in a release, I decided to create a separate PR to fix that, so that we can cherry-pick that change for the current documentation (see https://github.com/docker/cli/pull/3449)